caffeine@patapon.info/preferences/displayPage.js

```diff
@@ -70,10 +70,15 @@ class CaffeineDisplayPage extends Adw.PreferencesPage {
         });
 
         // Notifications
-        let notificationRow = new Adw.SwitchRow({
-            title: _('Notifications'),
-            subtitle: _('Enable notifications when Caffeine is enabled or disabled'),
-            active: this._settings.get_boolean(this._settingsKey.SHOW_NOTIFICATIONS)
+        let showNotificationToastRow = new Adw.SwitchRow({
+            title: _('Toast notification'),
```

Comment:
```suggestion
            title: _('State notification'),
```

A toast notification could be for anything, I think mentioning what the notification is for is better here